app/src/main/java/net/squanchy/support/widget/CardLayout.kt

```diff
@@ -0,0 +1,92 @@
+package net.squanchy.support.widget
+
+import android.content.Context
+import android.content.res.TypedArray
+import android.graphics.Outline
+import android.support.annotation.Px
+import android.util.AttributeSet
+import android.view.View
+import android.view.ViewOutlineProvider
+import android.widget.FrameLayout
+
+import net.squanchy.R
+
+open class CardLayout @JvmOverloads constructor(
+        context: Context,
+        attrs: AttributeSet,
+        defStyleAttr: Int = R.attr.cardViewDefaultStyle,
+        defStyleRes: Int = 0
+) : FrameLayout(context, attrs, defStyleAttr, defStyleRes) {
+
+    init {
+        super.setClipToOutline(false)
+
+        val a = context.obtainStyledAttributes(attrs, R.styleable.CardLayout, defStyleAttr, defStyleRes)
```

Comment:
We could actually use `use()`? https://kotlinlang.org/docs/reference/idioms.html#java-7s-try-with-resources